integration-tests/src/bin/exit_one.rs

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Right now, all these binaries will be built and installed during a `cargo install`. We probably don't want that. I think we can stop that from happening by putting `#![cfg(test)]` here.

Reply:
After @jyotsna-penumaka tried it, I confirmed this doesn't work. Resolved.